mecommerce/return-process
# fix(NotRefundedReason): update reason description for refunds on bundle breaking not supported. mecommerce/feature/CXL-25

# Overview of the changes
Fix description reason for refunds with bundle breaking not supported.

### Jira Link
https://thirdlove.atlassian.net/browse/CXL-25

# Steps for testing
1. checkout the branch on your local env.
2. Run an integration test which covers a not supported bundle breaking case.
3. Check either on your console logs or heaps events that the event is trigger with the reason "Bundle breaking not supported"
